localisation/000_wc_military_localization.csv

```diff
@@ -0,0 +1,470 @@
+wc_ret_rel_scourge2;Frost Wyrms;;Frost-W�rmer;;;;;;;;;;;x
+wc_ret_rel_scourge3;Flesh Titans;;Fleisch-Titanen;;;;;;;;;;;x
+wc_ret_rel_scourge4;Death Knights;;Todesritter;;;;;;;;;;;x
+wc_ret_rel_shaytar;Mindless Slaves;;Gedankenlose Sklaven;;;;;;;;;;;x
+wc_ret_rel_shaytar2;Cultists;;Kultisten;;;;;;;;;;;x
+wc_ret_rel_shaytar3;Faceless Reavers;;Gesichtslose Peiniger;;;;;;;;;;;x
+wc_ret_rel_shaytar4;Mutants;;Mutanten;;;;;;;;;;;x
+wc_ret_rel_legion;Lesser Demons;;Niedere D�monen;;;;;;;;;;;x
+wc_ret_rel_legion2;Greater Demons;;H�here D�monen;;;;;;;;;;;x
+wc_ret_rel_legion3;Fel Reavers;;Fel-Peiniger;;;;;;;;;;;x
+wc_ret_rel_legion4;Doomguard;;Unheilsh�ter;;;;;;;;;;;x
+wc_ret_rel_holylight;Paladins;;Paladine;;;;;;;;;;;x
+wc_ret_rel_cultofloa;Loa Host;;Loa-Heerschar;;;;;;;;;;;x
+wc_ret_rel_cultofloa2;Avatars;;Avatare;;;;;;;;;;;x
+wc_ret_rel_illidari;Demonhunters;;D�monenj�ger;;;;;;;;;;;x
+wc_ret_rel_kaldoreireligion;Animal Host;;Tierische Heerschar;;;;;;;;;;;x
+wc_ret_rel_kaldoreireligion2;Ancients;;Urtume;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+# Retinues � Nomadic;;;;;;;;;;;;;;x
+wc_ret_horde_ani;Raiders;;R�uber;;;;;;;;;;;x
+wc_ret_horde_cav;Outriders;;Vorreiter;;;;;;;;;;;x
+wc_ret_horde_ran;Marauders;;Marodeure;;;;;;;;;;;x
+wc_ret_horde_cas;Windchasers;;Windj�ger;;;;;;;;;;;x
+wc_ret_horde_lrg;Battle Lords;;Kampfherren;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+#Phase Descriptions;;;;;;;;;;;;;;x
+COMBATVIEW_PHASE_DESC_PURSUE;In the Pursue phase, one of the armies has routed and the other one is chasing them down and inflicting casualties.\n�YCavalry�W, �YAerial Infantry�W, �YDragons�W, �YElite Infantry�W and �YArchmages�W excel in the �YPursue Phase�W!;;In der Verfolgungsphase ist eine der Armeen geflohen und die andere verfolgt sie und f�gt Verluste zu.\n�YKavallerie�W, �YLufteinheiten�W, �YDrachen�W, �YEliteinfanterie�W und �YErzmagier�W gl�nzen in der �YPVerfolgungsphase�W!;;;;;;;;;;;x
+COMBATVIEW_PHASE_DESC_SKIRMISH;In the Skirmish phase, both armies will stay at a distance and attack with ranged missiles and magic.\n�YRanged Infantry�W, �YCasters�W, �YArchmages�W and �YSiege Machines�W excel in the �YSkirmish Phase�W!;;In der Scharm�tzelphase bleiben beide Armeen auf Distanz und greifen mit Fernwaffen und Magie an.\n�YFernk�mpfer�W, �YZauberer�W, �YErzmagier�W und �YBelagerungsmaschinen�W gl�nzen in der �YScharm�tzelphase�W!;;;;;;;;;;;x
+COMBATVIEW_PHASE_DESC_MELEE;In the Melee phase, both armies are locked in close combat until one is destroyed or routs.\n�YInfantry�W, �YCavalry�W, �YElite Infantry�W, �YLarge Cavalry�W, �YLarge Infantry�W, �YUndead�W, �YSwarm�W, �YThralls�W, �YAnimals�W, and �Ywar Machines�W excel in the �YMelee Phase�W!;;In der Nahkampfphase sind beide Armeen im Nahkampf gebunden, bis eine zerst�rt wird oder fl�chtet.\n�YInfanterie�W, �YKavallerie�W, �YElite-infanterie�W, �YGro�e Kavallerie�W, �YGro�e Infanterie�W, �YGhouls�W, �YSchw�rme�W, �YLeibeigene�W, �YTiere�W und �YKriegsmaschinen�W gl�nzen in der �YNahkampfphase�W!;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
```

Comment:
Also, horse archers are back, but loc about them in combat phases is not

Reply:
Added them back.